tst/standard/ReducibleMatrixGroups.tst

```diff
@@ -74,6 +74,26 @@ Error, <d> must be even
 gap> SpStabilizerOfNonDegenerateSubspace(4, 2, 3);
 Error, <k> must be less than <d> / 2
 
+#
+gap> TestOmegaStabilizerOfIsotropicSubspace := function(epsilon, d, q, k)
+>   local G;
+>   G := OmegaStabilizerOfIsotropicSubspace(epsilon, d, q, k);
+>   Assert(0, CheckSize(G));
+>   Assert(0, IsSubsetOmega(epsilon, d, q, G));
```

Comment:
I recommend running CheckSize last (see also PR #108; note that this PR effectively conflicts with your PR, as I renamed `IsSubsetOmega` and changed how it is used, see there)